libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1635,6 +1641,20 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         routeRefreshController.unregisterRouteRefreshStateObserver(routeRefreshStatesObserver)
     }
 
+    @ExperimentalPreviewMapboxNavigationAPI
```

Comment:
Is there a plan to make it a stable feature or it's designed for internal usage only?

Reply:
It will be stable. I just added this annotation for now in case we want to break API or sth.